sdk/cosmos/azure-spring-data-cosmos-test/src/test/resources/application.properties

```diff
@@ -7,6 +7,8 @@ dynamic.collection.name=spel-property-collection
 cosmos.queryMetricsEnabled=true
 # Max Degree of Parallelism allowed
 cosmos.maxDegreeOfParallelism=0
+# Max number of items to buffer
+cosmos.maxBufferedItemCount=-1
```

Comment:
We should set it to 0 (since that is the default), to avoid any false impressions on users when they look at this for reference, same for below.